### ISSUE TITLE
Documentation plans?

### ISSUE THREAD

**Issue by deliciouslytyped** [rui314/mold#3]
This is related to issue #2, 

Do you have any plans for documentation? (perhaps once the project is less in flux?)
An Issue I have often is that linker man pages are unclear or lacking. (once again, the fault may just be with me.)

It would be nice to have a well architected linker with good documentation. :)

**Comment by rui314** [comment]
Yes, I want to write a good documentation, but that's not a priority at this moment.
